lib/travis/api/v3/renderer/build.rb

```diff
@@ -14,7 +14,7 @@ def jobs
     end
 
     private def include_full_jobs?
-      return true if include?('build.job'.freeze) || include?('build.jobs'.freeze)
+      return true if include?('build.jobs'.freeze)
```

Comment:
Interesting – I left the singular `job` in because it was already there. I assumed it was needed somewhere other than the endpoint I was working on, but maybe not...?